etna/auto/auto.py

```diff
@@ -37,7 +39,152 @@ def __call__(self, pipeline: Pipeline) -> None:
         ...
 
 
-class Auto:
+class AutoBase:
```

Comment:
I think we should inherit this class from `AutoAbstract` because it partially implements this interface.